turbinia/workers/analysis/auth.py

```diff
@@ -42,9 +76,13 @@ def __init__(self):
     self.last_seen = 0
 
     # The first time the source_ip or username successfully login.
-    self.first_auth_timestamp = 0
-    self.first_auth_ip = ''
-    self.first_auth_username = ''
+    self.first_auth = None
```

Comment:
What data type is this attribute?  String? Datetime?  Should we add an `Attributes` section to the class docstring?

Reply:
Fixed - added `Attributes`.